app/storybook/documentation/ui-kit/QueryHooks.mdx

```diff
@@ -0,0 +1,51 @@
+import { ArgTypes, Meta } from '@storybook/blocks'
+import LinkTo from '@storybook/addon-links/react'
+import { SourceCode } from '../../.storybook/blocks/SourceCode'
+import * as useCounterQueryStories from '../../../../packages/ui-kit/src/hooks/useCounter/useCounter.stories'
+import * as CounterQueryPropsStories from '../../../../packages/ui-kit/src/components/Counter/CounterQueryProps.stories'
+
+import * as useLeaderboardStories from '../../../../packages/ui-kit/src/hooks/useLeaderboard/useLeaderboard.stories'
+import * as LeaderboardQueryPropsStories from '../../../../packages/ui-kit/src/components/Leaderboard/LeaderboardQueryProps.stories'
+
+import * as useTimeSeriesStories from '../../../../packages/ui-kit/src/hooks/useTimeSeries/useTimeSeries.stories'
+import * as TimeSeriesQueryPropsStories from '../../../../packages/ui-kit/src/components/TimeSeries/TimeSeriesQueryProps.stories'
+
+<Meta title="Customization/Query Hooks" />
+
+# Query Hooks
+
+Propel’s UI Kit library provides the following React components: Counter, TimeSeries, and Leaderboard. To query the GraphQL API, we have developed 
+Query Hooks including "useCounter", "useLeaderboard" and "useTimeSeries.
+
+## useCounter
+
+<SourceCode of={useCounterQueryStories.FractionUnicode} shown />
+
+### `CounterQueryProps`
```

Comment:
From my understanding, this document provides an overview of the Query Hooks. I suggest moving the prop tables to the API section and replacing them with links in this document to reduce its size. Wdyt?